### ISSUE TITLE
No color output

### ISSUE THREAD

**Issue by hnrw** [XAMPPRocky/tokei#830]
Sorry if I'm missing something obvious, but I've spent quite a while trying to turn on colored output, but everytime it comes out black and white.

![image](https://user-images.githubusercontent.com/61143948/133857484-de92b010-c3c3-47cc-905d-7ed4f5288717.png)

I don't have NO_COLOR set, and other programs print with color fine.

Since tokei uses colored, I tried testing colored in a separate project. And it works just fine.

```
use colored::Colorize;
fn main() {
    let testing = "this is blue".blue();
    println!("{}", testing);
}
```
![image](https://user-images.githubusercontent.com/61143948/133857900-9a0cffe5-c989-4b6e-aa89-f542b5aa2808.png)

Am I missing an obvious flag?

I'm running WSL2 in Windows 10. Ubuntu 20.04. Installed tokei with cargo.

**Comment by XAMPPRocky** [comment]
Thank you for your issue! Is this also an issue if you compile from source using something like `cargo install --git https://github.com/XAMPPRocky/tokei.git`?

**Comment by hnrw** [comment]
Colors works correctly when compiling from source.

That solves the issue for me, but I don't know why. I'm happy to mark this closed unless you want me to test see if I can nail the problem?

**Comment by matu3ba** [comment]
@hnrw Did you use the corresponding tag for the checkout (v12.1.2)? If yes: Can you check, if files are identical of the tag to the crates.io release? You can use [cargo download](https://github.com/JanLikar/cargo-clone) and git clone hash, delete all the git clutter and run a simple recursive diff to compare the files.

**Comment by XAMPPRocky** [comment]
@hnrw Yeah, if it works from source, there's no much I can do. I haven't released a new version in a while, so there's been a lot of change between what's on GitHub and what's on crates.io

**Comment by kyoheiu** [comment]
Seems like old issue, but I have the similar issue: With Arch Linux version, tokei does not give a colorized output, but compiling it from GitHub source solves this issue. 
(Each language name is not colorized even by GitHub version, but this is correct behavior I think)

**Comment by Kuba-Jasinski** [comment]
Unfortunately, as of 2025, the issue persists in Arch Linux (when installed from AUR).
